packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -66,8 +65,37 @@ class BoundingVolumeHierarchy
   private:
     friend struct Details::TreeTraversal<DeviceType>;
 
-    Kokkos::View<Node *, DeviceType> _leaf_nodes;
-    Kokkos::View<Node *, DeviceType> _internal_nodes;
+    Kokkos::View<Node *, DeviceType> getInternalNodes()
+    { assert( !empty() ); return Kokkos::subview( _internal_and_leaf_nodes, std::make_pair(size_type{0}, size() - 1) ); }
```

Comment:
I think that this function should not assert even if the tree is empty. Have you tried removing assert? It may just return an empty view with boundaries [0, -1).